tests/behaviour/expectations/behaviour.ts

```diff
@@ -88,6 +88,15 @@ export const expectations = flatten(
               Expect.Simple('getIntValue', [], ['512'], '0'),
               Expect.Simple('addUintValue', ['0', '256'], ['0', '512'], '0'),
               Expect.Simple('addIntValue', ['256'], ['768'], '0'),
+              new Expect('testing constructor argument out of bounds', [
```

Comment:
If we're using Expects specifically for failing constructor tests, I think it's worth putting something into the `Expect` constructor in `types.ts` that checks whether any of the function names in `steps` is `constructor` and if so, it should assert that the expected return value for that `Expect` is null